packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/flavors-list.html

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+    <oui-select-picker
+        class="d-inline-block col-md-6 col-xl-4 my-3"
+        data-ng-repeat="flavor in $ctrl.flavors track by flavor.name"
+        data-name="{{:: flavor.name }}"
+        data-model="$ctrl.selectedFlavor"
+        data-label="{{:: $ctrl.capitalize(flavor.name) }}"
+        data-values="[flavor]"
+        data-variant="light"
+        data-required
+    >
+        <oui-select-picker-section>
+            <div class="d-flex">
+                <span
+                    data-translate="pci_database_flavors_list_spec_ram"
+                    data-translate-values="{ 
+                        ram: (flavor.memory | cucBytes:0:false:'GB')
+                    }"
+                ></span>
+            </div>
+            <div class="d-flex">
+                <span
+                    data-translate="pci_database_flavors_list_spec_core"
+                    data-translate-values="{ cores: flavor.core }"
+                ></span>
+            </div>
+            <div class="d-flex">
+                <span
+                    data-translate="pci_database_flavors_list_spec_storage"
+                    data-translate-values="{ 
+                    storage: (flavor.storage | cucBytes:2:false:'GB')
+                }"
+                ></span>
+            </div>
+        </oui-select-picker-section>
+        <oui-select-picker-section>
+            <ovh-manager-catalog-price
+                data-price="flavor.monthlyPrice.priceInUcents"
+                data-tax="flavor.monthlyPrice.tax"
+                data-user="$ctrl.user"
+                data-show-zero-price="true"
```

Comment:
It could have been nice to allow this syntax
```suggestion
                data-show-zero-price
```